index.js

```diff
@@ -1,265 +1,226 @@
 'use strict';
 
-module.exports = function(S) {
-  const path     = require('path'),
-    SError       = require(S.getServerlessPath('Error')),
-    SCli         = require(S.getServerlessPath('utils/cli')),
-    BbPromise    = require('bluebird'),
-    async        = require('async'),
-    _            = require('lodash'),
-    mime         = require('mime'),
-    fs           = require('fs');
-
-  class ClientDeploy extends S.classes.Plugin {
-
-    constructor() {
-      super();
-      this.name = 'serverless-client-s3';
-    }
-
-    registerActions() {
-      S.addAction(this.clientDeploy.bind(this), {
-        handler:       'clientDeploy',
-        description:   `Deploy your Serverless clients to S3 Website Bucket.`,
-        context:       'client',
-        contextAction: 'deploy',
-        options:       [
-          {
-            option:      'stage',
-            shortcut:    's',
-            description: 'stage to populate any variables'
-          }, {
-            option:      'region',
-            shortcut:    'r',
-            description: 'region to populate any variables'
+const path     = require('path');
+const BbPromise    = require('bluebird');
+const async        = require('async');
+const _            = require('lodash');
+const mime         = require('mime');
+const fs           = require('fs');
+const AWS = require('serverless/lib/plugins/aws');
```

Comment:
This feels like it should be exposed on the `serverless` object like `utils` and `cli`